cmd/hz/config/argument.go

```diff
@@ -228,12 +228,12 @@ func (arg *Argument) checkPackage() error {
 			if err != nil {
 				return fmt.Errorf("can not get relative path, err :%v", err)
 			}
-			module = filepath.Join(module, rel)
-			arg.Gomod = module
+			arg.Gomod = filepath.Join(module, rel)
 		}
 		if len(arg.Gomod) == 0 {
 			arg.Gomod = arg.Gopkg
 		}
+		arg.Gomod = util.PathToImport(module, "")
```

Comment:
Seems that line 233~235 become meaningless according to this change?

Reply:
done